src/v/kafka/server/group.cc

```diff
@@ -2338,14 +2338,22 @@ group::handle_offset_fetch(offset_fetch_request&& r) {
         for (const auto& e : _offsets) {
             offset_fetch_response_partition p = {
               .partition_index = e.first.partition,
```

Comment:
```
kafka/server: Check pending transaction for fetch_offset

Client should retry fetch_offset request if
partitions inside group have pending transactions
```

Please explain what this commit is doing.

1. What does "Check pending transaction for fetch_offset" mean
2. What changes in this commit cause "Client should retry fetch_offset" to occur

Is `unstable_offset_commit` part of how this works and a signal to clients? That should be mentioned some where.

Reply:
If client (I mean client library) set require_stable, redpanda should return error, if some transactions work with partition from this request. After it client library will retry this request

Reply:
Client - person, should not do anything additional. It is only client library logic

Reply:
Please explain it in the commit message